README.md

```diff
@@ -101,7 +103,7 @@ https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#stub-versioning).
 ## Contributing
 
 Please read [CONTRIBUTING.md](CONTRIBUTING.md) before submitting pull
-requests.
+requests. If you have questions related to contributing, drop by the [mypy Gitter](https://gitter.im/python/mypy).
```

Comment:
Actually the plan is to completely kill the python.mypy gitter room, so this should also point to python/typing.